tutorials/Understanding_VISSL_Training_and_YAML_Config.ipynb

```diff
@@ -266,7 +266,11 @@
         "\n",
         "Next step for us is to register the dummy data we created above with VISSL. Registering the dataset involves telling VISSL about the dataset name and the paths for the dataset. For this, we create a simple json file with the metadata and save it to `configs/config/dataset_catalog.py` file.\n",
         "\n",
-        "**NOTE**: VISSL uses the specific `dataset_catalog.json` under the path `configs/config/dataset_catalog.json`."
+        "**NOTE**: VISSL uses the specific `dataset_catalog.json` under the path `configs/config/dataset_catalog.json`.\n",
+        "\n",
+        "**NOTE**: If you want to preserve the older entries and create a newline delimited json for `dataset_catalog.json`, use `append_to_json=True` in the `save_file` function. \n",
```

Comment:
Sorry, I don't think I was super clear in my comments. new_line_delimited json won't work for the dataset_catalog -- only json, we have to use append_to_json=False. I personally don't think we need to add a comment here, as this is really just an implementation detail. 

What I meant by new_line_delimited json was when we save the metrics.json file. See: https://github.com/facebookresearch/vissl/blob/master/vissl/hooks/log_hooks.py#L499
which yields the following format:

```
{"iteration": 312, "phase_idx": 0, "train_phase_idx": 0}
{"iteration": 624, "phase_idx": 1, "train_phase_idx": 1}
{"iteration": 936, "phase_idx": 2, "train_phase_idx": 2}
{"iteration": 1248, "phase_idx": 3, "train_phase_idx": 3}
{"iteration": 1560, "phase_idx": 4, "train_phase_idx": 4}
{"iteration": 1872, "phase_idx": 5, "train_phase_idx": 5}
```

Reply:
Addressed both the comments in e6449ba